### DISCUSSION TITLE
ConnectButton

### DISCUSSION THREAD

**Discussion by AbuSantos** [web3ui/web3uikit#765] [Q&A] [answered]
I keep getting this error after I Import the connectButton from web3uikit,

Module not found: Can't resolve 'web3uikit' 
| import { ConnectButton } from "web3uikit";  

This is my package json file
```
    "dependencies": {
      "@web3uikit/core": "^0.1.3",
      "@web3uikit/icons": "^0.1.3",
      "@web3uikit/web3": "^0.1.3",
      "moralis": "^1.10.1",
      "next": "12.2.3",
      "react": "18.2.0",
      "react-dom": "18.2.0",
      "react-moralis": "^1.4.0",
      "web3uikit": "^1.0.2"
```

**Comment by KMean** [comment, marked as answer]
should be:
import { ConnectButton } from "@web3uikit/web3"

**Comment by AbhinavMV** [comment]
Hey @AbuSantos, Please update to the latest version 0.1.5
Then use the component like @KMean has shown 
TY @KMean 🙌🏻
